metrics/metrics.go

```diff
@@ -0,0 +1,25 @@
+package metrics
+
+import (
+	"github.com/go-kit/kit/metrics"
+	"github.com/go-kit/kit/metrics/prometheus"
+	stdprometheus "github.com/prometheus/client_golang/prometheus"
+)
+
+//Provider is intended to be a metrics API for webpa servers.
+type Provider struct {
```

Comment:
We don't want our own Provider type.  We want to use go-kit's `metrics.provider.Provider` interface and associated types.  The first version can simply use `metrics.provider.NewPrometheusProvider`.